src/charmed_kubeflow_chisme/juju/_juju.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+"""Helper for shelling out to the Juju CLI"""
+
+from subprocess import Popen, PIPE
+from ruamel.yaml import YAML
+
+
+class Juju:
```

Comment:
I was just wondering. This looks like a handy function which is  also useful for people outside kubeflow team isnt there a lib from juju team providing us this capability (I mean executing commands and getting info)

Reply:
yeah I feel the same way.  If anywhere, I think it would be in [python libjuju](https://pythonlibjuju.readthedocs.io/en/latest/), but I didn't see it there.  I'll ask around